cmd/cli/serve/serve.go

```diff
@@ -173,6 +174,11 @@ func serve(cmd *cobra.Command) error {
 		return err
 	}
 
+	startWebUI, err := config.Get[bool](types.NodeWebUI)
+    if err != nil {
+        return err
+    }
+
```

Comment:
Formattings a bit wonky.  If your editor supports plugins, you can get them to run `go fmt` for you whenever it saves a file, and that'll save you having to indent stuff yourself.